styles/styles.css

```diff
@@ -358,6 +381,243 @@ h2.advantages__heading:after {
   stroke-width: 2;
 }
 
+/* DESTINATIONS */
+.destinations {
+  max-width: 1440px;
+  max-height: 958px;
+  height: inherit;
+  padding-top: 120px;
+  padding-bottom: 208px;
+  margin: 0 auto;
+}
+
+.destinations__tabs {
+  width: inherit;
+  max-width: calc(192px + 192px + 238px);
+  height: 64px;
+  margin-bottom: 65px;
+  display: flex;
+  position: relative;
+  z-index: 10;
+}
+
+.destinations__tab_regions {
+  flex-basis: 192px;
+}
+
+.destinations__tab_cities {
+  flex-basis: 192px;
+}
+
+.destinations__tab_poi {
+  flex-basis: 238px;
+}
+
+.destinations__tabs::after {
+  position: absolute;
+  display: block;
+  width: 100%;
+  height: inherit;
+  content: "";
+  background-color: var(--stroke-color-main);
+  border-radius: 8px;
+  z-index: -1;
+}
+
+.destinations__tab {
+  background-color: var(--stroke-color-main);
+  font-size: 24px;
+  text-decoration: none;
+  text-align: center;
+  padding: 17px 0;
+  border-radius: 8px;
+}
+
+.destinations__tab:link, .destinations__tab:visited {
+  color: var(--text-color-main);
+}
+
+.destinations__tab:hover {
+  box-shadow: inset 0 0 0 3px var(--accent-blue);
+}
+
+.destinations__tab._active, .destinations__tab:active {
+  background-color: var(--accent-blue);
+  color: var(--general-white);
+}
+
+.destinations__list {
+  width: fit-content;
+  max-width: calc(1232px + 32px);
```

Comment:
The same

Reply:
Fixed 👉 https://github.com/pavel-4/if-html/pull/12/commits/2c1cf0dc0200b9ce9ffb87cffed1a582b9dd3cc5 cc @almuly